saleor/tests/e2e/orders/test_delete_draft_order.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..product.utils.preparing_product import prepare_product
+from ..shop.utils.preparing_shop import prepare_shop
+from ..utils import assign_permissions
+from .utils import (
+    draft_order_create,
+    draft_order_delete,
+    draft_order_update,
+    order_lines_create,
+    order_query,
+)
+
+
+@pytest.mark.e2e
+def test_delete_draft_order_CORE_0205(
+    e2e_staff_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+    permission_manage_orders,
+):
+    # Before
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_product_types_and_attributes,
+        permission_manage_orders,
+    ]
+    assign_permissions(e2e_staff_api_client, permissions)
+
+    price = 10
+
+    (
+        result_warehouse_id,
+        result_channel_id,
+        _,
+        result_shipping_method_id,
+    ) = prepare_shop(e2e_staff_api_client)
+
+    _, result_product_variant_id, _ = prepare_product(
+        e2e_staff_api_client, result_warehouse_id, result_channel_id, price
+    )
```

Comment:
I recommend putting a name to unused variables so maintainers can instantly know what the unused variable contains, e.g.:

```py
# avoid
instance, _ = Model.objects.get_or_create(...)

# prefer
instance, _created = Model.objects.get_or_create(...)

Reply:
Nice to know! I will do refactor of the code with the suggested change in a separate PR.